src/main/java/org/aion/wallet/connector/WalletBlockchainConnector.java

```diff
@@ -0,0 +1,73 @@
+package org.aion.wallet.connector;
+
+import org.aion.api.server.ApiAion;
+import org.aion.api.server.types.ArgTxCall;
+import org.aion.base.type.Address;
+import org.aion.wallet.WalletApi;
+import org.aion.wallet.connector.dto.SendRequestDTO;
+import org.aion.wallet.connector.dto.UnlockableAccount;
+import org.aion.zero.impl.types.AionBlock;
+import org.aion.zero.types.AionTransaction;
+
+import java.util.ArrayList;
+import java.util.List;
+
+//todo: add validation and errors
+public class WalletBlockchainConnector implements BlockchainConnector {
+    // todo: will we be able to access this from AccountManager?
+    private static final Integer DEFAULT_UNLOCK_DURATION = 1000;
+
+    private ApiAion aionApi = new WalletApi();
```

Comment:
I'm weondering ig this should not be `private static final` since we don't need a new WalletApi even if we would instantiate more of `WalletBlockchainConnector`